AEPEdge.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "AEPEdge"
-  s.version          = "1.0.0-beta.2"
+  s.version          = "1.0.0"
```

Comment:
@shalehaha @nporter-adbe I think this podspec should declare a dependency on ` s.dependency 'AEPCore', '>= 3.0' ` 
also, let's update swift version to 5.1 here as well

Reply:
@emdobrin I updated the podspec to 5.1 for the swift version, but I'm not seeing the point to enforce >= 3.0 as there will never be a version less than 3.0 for AEPCore.

Reply:
I think it is safe we can remove AEPServices since it is part of the AEPCore spec

Reply:
updated.